packages/itwinui-react/src/core/Tooltip/Tooltip.tsx

```diff
@@ -175,7 +175,9 @@ const useTooltip = (options: TooltipOptions = {}) => {
       handleClose: safePolygon({ buffer: -Infinity }),
     }),
     useFocus(floating.context),
-    useClick(floating.context),
+    useClick(floating.context, {
```

Comment:
Do we even need `useClick` here? What exactly is it doing for us?

Reply:
I tried to not remove it as far as possible because I assumed that it was there for a reason. But I feel having just hover and focus as triggers are sufficient in this case.

So, I removed `useClick` (https://github.com/iTwin/iTwinUI/pull/1944/commits/66d1900841727d5e4fdd8d0da2b1c87d0c8b8c2a). Removing `useClick` seems to be giving the same behavior as `useClick({ ignoreMouse: true })`, considering that the focus and hover triggers are also present and unchanged.

Reply:
Actually I just realized what `useClick` is doing for us: it's hiding the tooltip when the button is clicked.

So it's not needed for the purposes of _triggering_ (showing) the tooltip on click, but it's needed for hiding it.

This makes sense from a UX perspective — the user will first hover (or focus) the button to understand what it does, and then they will click (or press <kbd>Enter</kbd>) to trigger the action. At this point, they no longer need to see the tooltip.

Reply:
`referencePress` seems to dismiss upon click but not upon focus + <kbd>space</kbd>. `useClick({ ignoreMouse: true })` + `useDismiss({})` dismisses upon focus + <kbd>space</kbd> but not upon click.

This is why in [`f365335` (#1944)](https://github.com/iTwin/iTwinUI/pull/1944/commits/f3653355296b6710890befc2bd10825227bdde9a), I used both `ignoreMouse` in `useClick` and `referencePress` in `useDismiss`.

The issue now is that pressing <kbd>space</kbd> repeatedly upon focusing toggles the tooltip. But clicking repeatedly just closes the tooltip upon the first click and does not bring it back the second click onwards. Not sure if this is an issue:

https://github.com/iTwin/iTwinUI/assets/45748283/444f2134-ab35-40e1-9ebd-6137a772295a

Reply:
[floating-ui docs](https://floating-ui.com/docs/usedismiss#keyboard-press-dismissal) suggest adding a custom `onClick` (which is called on pressing <kbd>Enter</kbd>/<kbd>Space</kbd> when used on `<button>`). We could do that to avoid toggling repeatedly.

Reply:
Alternatively we could just change [`referencePressEvent`](https://floating-ui.com/docs/usedismiss#referencepressevent) to `click`?

Reply:
Earlier, in [`f365335` (#1944)](https://github.com/iTwin/iTwinUI/pull/1944/commits/f3653355296b6710890befc2bd10825227bdde9a), I used both `ignoreMouse` in `useClick` and `referencePress` in `useDismiss`.

The issue it gave was that pressing <kbd>space</kbd> repeatedly upon focusing toggles the tooltip. But clicking repeatedly just closes the tooltip upon the first click and does not bring it back the second click onwards.

https://github.com/iTwin/iTwinUI/assets/45748283/444f2134-ab35-40e1-9ebd-6137a772295a

But now, in [`1ae6d27` (#1944)](https://github.com/iTwin/iTwinUI/pull/1944/commits/1ae6d27b904fe4869dca1a5a8c5260a4816dc9eb), using `useDismiss({ referencePress: true, referencePressEvent: "click" })` + removing `useClick()` seems to give consistent behavior regardless of mouse or keyboard clicks.

So now, clicking just once using a mouse or a keyboard dismisses the tooltip.

https://github.com/iTwin/iTwinUI/assets/45748283/11d62b33-fd00-4544-a434-7fe9121e89ed

I also added `move: false` in `useHover` to prevent mouse movements within the trigger region from re-triggering the tooltip.

<details>
<summary>Screen recordings</summary>

Without `move: false`:

https://github.com/iTwin/iTwinUI/assets/45748283/5ea5311d-3f23-4b08-a23b-93a881a0bebc

With `move: false`:

https://github.com/iTwin/iTwinUI/assets/45748283/aff213de-735d-4bda-b979-fa1c0d0307dd

</details>